### ISSUE TITLE
Changed interface in "Actions console" "Smart Home"  replace by something new in Developer console

### ISSUE THREAD

**Issue by tonsum** [home-assistant/home-assistant.github.io#35902]
### Feedback

Changed interface in "Actions console" "Smart Home” seems om be replaced by something more complicated in Developer console. It´s now hard to follow the guideline. I think it would be much appreciated if it was updated to current conditions.  :-) Best regards Torbjorn

### URL

https://www.home-assistant.io/integrations/google_assistant/

### Version

2024.11.2

**Comment by y8s** [comment]
The Actions console is going away and will be replaced. See here: https://github.com/home-assistant/home-assistant.io/issues/35867
